doc/tutorial/climada_entity_LitPop.ipynb

```diff
@@ -405,7 +405,7 @@
     "exp_population = LitPop().from_population(countries=country, res_arcsec=res)\n",
     "exp_population.plot_hexbin(linewidth=markersize, buffer=buffer_deg);\n",
     "# Compare to default LitPop exposures:\n",
-    "exp = LitPop.from_countries('JAM', res_arcsec=res)\n",
+    "exp = LitPop.from_countries(countries = country, res_arcsec=res)\n",
```

Comment:
```suggestion
    "exp = LitPop.from_countries(countries=country, res_arcsec=res)\n",
```

Reply:
Thank you! But somehow I couldn't commit it: It's said "This diff has recently been updated." and refreshing doesn't help.

Reply:
same here. don't know what's up. I've changed it from local reposiory now.